PersonDataRequest/PersonDataRequest/Controller/ApiHelper.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  ApiHelper.swift
+//  PersonDataRequest
+//
+//  Created by Ivan on 13.02.2024.
+//
+
+import Foundation
+
+struct ApiHelper {
+    
+    func makeRequest<T: Decodable>(urlString: String, t: T.Type, completion: @escaping (T?) -> Void) {
+        guard let url = URL(string: urlString) else {
+            completion(nil)
+            return
+        }
+        
+        let task = URLSession.shared.dataTask(with: url) { (data, response, error) in
+            if let error = error {
+                print("Gender request error: \(error.localizedDescription)")
+                completion(nil)
```

Comment:
It is the wrong one, you needed to remove the return OR add completion(nil)
So it should be:
```
if let error = error {
                print("Gender request error: \(error.localizedDescription)")
                completion(nil)
                return
 }
```
OR

```
if let error = error {
                print("Gender request error: \(error.localizedDescription)")
}
```